PyInstaller/utils/hooks/qt.py

```diff
@@ -618,7 +628,14 @@ def _find_all_or_none(globs_to_include, num_files, qt_library_info):
         )
         dll_file_paths = glob.glob(dll_path)
         for dll_file_path in dll_file_paths:
-            to_include.append((dll_file_path, dst_dll_path))
+            dll_file_path = pathlib.Path(dll_file_path).resolve()
```

Comment:
Based on earlier discussion about `Path.resolve()`, I've now added the call here as well. I don't think it should make much difference, but it makes the changes in the PR consistent (i.e., all added `pathlib.Path()` instances are explicitly resolved now).